src/node.cc

```diff
@@ -1399,6 +1399,17 @@ ssize_t DecodeWrite(Isolate* isolate,
   return StringBytes::Write(isolate, buf, buflen, val, encoding, nullptr);
 }
 
+bool IsExceptionDecorated(Environment* env, Local<Value> er) {
+  if (!er.IsEmpty() && er->IsObject()) {
+    Local<Object> err_obj = er.As<Object>();
+    Local<Value> decorated = err_obj->GetHiddenValue(env->decorated_string());
+    return (!decorated.IsEmpty() &&
+            decorated->IsBoolean() &&
+            decorated->BooleanValue());
```

Comment:
Superfluous parentheses and you can shorten it to `return !decorated.IsEmpty() && decorated->IsTrue()`, which should also be infinitesimally faster.